src/pykeen/models/unimodal/trans_r.py

```diff
@@ -81,38 +81,33 @@ def __init__(
             preferred_device=preferred_device,
             random_seed=random_seed,
             regularizer=regularizer,
+            entity_initializer=xavier_uniform_,
+            entity_constrainer=clamp_norm,
+            entity_constrainer_kwargs=dict(maxnorm=1., p=2, dim=-1),
+            relation_initializer=xavier_uniform_normed_,
```

Comment:
should we dispense with `xavier_uniform_normed_` and just use the `compose_` function to create it on the fly?

Reply:
Yes, I think this would be easier.